django/db/backends/postgresql/compiler.py

```diff
@@ -0,0 +1,49 @@
+from django.db.models.sql.compiler import (
+    SQLAggregateCompiler,
+    SQLCompiler,
+    SQLDeleteCompiler,
+)
+from django.db.models.sql.compiler import SQLInsertCompiler as BaseSQLInsertCompiler
+from django.db.models.sql.compiler import SQLUpdateCompiler
+
+__all__ = [
+    "SQLAggregateCompiler",
+    "SQLCompiler",
+    "SQLDeleteCompiler",
+    "SQLInsertCompiler",
+    "SQLUpdateCompiler",
+]
+
+
+class InsertUnnest(list):
+    """
+    Sentinel value to signal `DatabaseOperations.bulk_insert_sql` that the
+    UNNEST strategy should be used for the bulk insert.
```

Comment:
A link to something authoritative on the benefits might be a nice addition to this docstring for future maintainers 🤔

Reply:
I couldn't find any non-commercial source on the strategy (either SO, or the Timescale blog).

I searched both the [docs](https://www.postgresql.org/docs/) and the [wiki](https://wiki.postgresql.org/).

Any reason why the `git-blame` trace back to the ticket and forum discussion is not enough?